java/integrated-mesh-layer/README.md

```diff
@@ -0,0 +1,27 @@
+# Add an integrated mesh layer
+
+View an integrated mesh layer from a scene service.
+
+![Integrated Mesh Layer](integrated-mesh-layer.png)
+
+## Use Case
+
+An integrated mesh can represent artificial and natural 3D features such as
+building walls, trees, valleys, and cliffs, with realistic textures and
+elevation information.
+
+## How it works
+
+1. Create a scene.
+2. Create an `IntegratedMeshLayer` with the URL to an integrated mesh layer 
+scene service.
+3. Add the layer to the scene's operational layers collection.
+4. Create a `Camera` and set the `SceneView`'s viewpoint.
+
+## Relevant API
+
+- IntegratedMeshLayer
+
+## Tags
+
```

Comment:
This line needs to have just "Layers" on it. It's used to define the category of the sample.

It's an inelegant, but convenient way to easily change categories at the moment.

Reply:
I've just copied what the Design README defined, but shouldn't each plaform match as close as possible?

https://devtopia.esri.com/runtime/common-samples/blob/master/designs/AddAnIntegratedMeshLayer/README.md

Reply:
Tags are still a comparatively new thing in READMEs which haven't been fully sorted on all platforms. Just put the tags on the following lines. The Android sample viewer build script reads the first tag and uses it for the category

So:

#### Tags
Layers
3D
integrated mesh